scripts/generate_error_code.hs

```diff
@@ -129,8 +129,7 @@ internalErrors = errorsEnumerate 300
   , "internal_309"
       :? "Thrown when `(p.dx - r.dx)` is not nat."
 
-  , "internal_insufficient_balance_err"
-      :? "Contract does not have enough liquidity to execute the swap."
+  , removedError
```

Comment:
Can we not simply delete the entry from the list?

Reply:
I don't think so, the enumeration will make it so that the errors won't line up anymore both in their names (e.g. `internal_311` will have the code `410`) and in how they are matched up in the code.

Both could be changed of course, it's probably not worth the trouble right now tho.